### ISSUE TITLE
Error when importing [Java Projects] for [multi-module] maven projects

### ISSUE THREAD

**Issue by futuretelematics** [microsoft/vscode-java-dependency#691]
If the [vs-code] [workspace] contains **more than a single** folder like:  
```
{
	"folders": [
		{ 
			"path": "../projects/projectA"
		},
		{ 
			"path": "../projects/projectB"
		}
	]
}
```
and both [projectA] and [projectB] are **multi-module** maven projects like:  
```
[projectA]
    + [moduleA.1]
    + [moduleA.2]
[projectB]
    + [moduleB.1]
    + [moduleB.2]
```
The *expected* java projects at the [Java Projects] view should be:  
```
[Java Projects]
    + [projectA]
        + [moduleA.1]
        + [moduleA.2]
    + [projectB]
        + [moduleB.1]
        + [moduleB.2]
```  
BUT the *parent* maven projects are being *detected* as java projects and the [Java Projects] view is like:
```
[Java Projects]
    + [projectA]
        + [moduleA.1]
        + [moduleA.2]
        + [projectA]   <-- this is NOT a java project 
        + [projectA]   <-- this is NOT a java project 
    + [projectB]
        + [moduleB.1]
        + [moduleB.2]
        + [projectB]   <-- this is NOT a java project
        + [projectB]   <-- this is NOT a java project
```  

If the [vs-code] workspace contains a SINGLE folder:  
```
{
	"folders": [
		{ 
			"path": "../projects/projectA"
		}
	]
}
```  
then the [Java Projects] view is OK:
```
[Java Projects]
    + [moduleA.1]
    + [moduleA.2]
```  
(the *parent* project is NOT detected as a [java project])

Checked in [vs-code] insiders preview 1.72.0

**Comment by futuretelematics** [comment]
Attached you have a two TEST project
[test.zip](https://github.com/microsoft/vscode-java-dependency/files/9669137/test.zip)
In order to reproduce the issue:  
1) create [vs-code] workspace with JUST ONE project; you'll see that the "detected" [java projects] from the maven project are:
```  
[Java Projects]
    + [moduleA.1]
    + [moduleA.2]
```   
2) ADD the second project to the [vs-code] workspace and you'll see that the "detected" [java projects] form the maven projects are:  
```
[Java Projects]
    + [moduleA.1]
    + [moduleA.2]
    + [moduleB.1]
    + [moduleB.2]
    + [projectA]   <-- this is NOT a java project 
    + [projectB]   <-- this is NOT a java project
```
Here you have an screenshot of the situation:   
![screenshot](https://user-images.githubusercontent.com/146404/192896381-ef335c98-3092-40d2-9412-bf129cc6b9e5.png)

**Comment by jdneo** [comment]
I can repro this bug.

It's caused by https://github.com/microsoft/vscode-java-dependency/blob/e60b2b33dd26d42dca7d831461f764aafd75cbcd/jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/ProjectCommand.java#L115-L117

**Comment by jdneo** [comment]
The fix should be available in `0.21.2022102500`
